plugins/org.jkiss.dbeaver.ext.generic/plugin.xml

```diff
@@ -954,6 +954,38 @@
                     <parameter name="supports-truncate" value="false"/>
                 </driver>
 
+                <!-- TDengine -->
+                <driver
+                    id="tdengine"
+                    label="TDengine"
+                    class="com.taosdata.jdbc.rs.RestfulDriver"
+                    sampleURL="jdbc:TAOS-RS://{host}:{port}/[{db}]"
+                    defaultPort="6041"
+                    description="TDengine driver"
+                    supportedConfigurationTypes="MANUAL"
+                    webURL="http://www.tdengine.com"
+                    icon="icons/tdengine_icon.png"
+                    iconBig="icons/tdengine_icon_big.png"
+                    categories="timeseries">
+                    <file type="jar" path="https://repo1.maven.org/maven2/com/taosdata/jdbc/taos-jdbcdriver/3.2.1/taos-jdbcdriver-3.2.1-dist.jar"/>
```

Comment:
Please do not use direct maven repo URLs.
Use artifact references like
<file type="jar" path="maven:/taosdata:taos-jdbcdriver:RELEASE[XXX]"/>
similarly to all other artifact references in generic plugin.

Reply:
fixed